cmake/configs/default.cmake

```diff
@@ -62,7 +62,7 @@ hunter_config(Libcxxabi VERSION ${HUNTER_Clang_VERSION})
 hunter_config(Libssh2 VERSION 1.7.0)
 hunter_config(Lua VERSION 5.3.2)
 hunter_config(MySQL-client VERSION 6.1.6)
-hunter_config(OpenCV VERSION 3.1.0-p1)
+hunter_config(OpenCV VERSION 3.1.0-p2 CMAKE_ARGS WITH_QT=YES)
```

Comment:
> If one would chose to use a previous version in their local config that was not patched for the Qt backend, such as 2.4 or 3.0, the build would break

But same arguments can be applied for the opposite situation when I want to set compatible version and `WITH_QT=YES` will unexpectedly gone. I think leaving a note to the wiki is a reasonable choice. Also you can check version in `hunter.cmake`, see for example Qt:
* https://github.com/ruslo/hunter/blob/d6177f64253d19c3853a3511a8af82ea99d2e145/cmake/projects/Qt/hunter.cmake.in#L18